include/mbedtls/ssl.h

```diff
@@ -167,19 +167,19 @@
 #define MBEDTLS_ERR_LAST 0x7F80 /**< This definition points to the last error code to have a correct parsing in error.c */
 
  /* List of extensions used in ssl_internal.h / extensions_present in mbedtls_ssl_handshake_params */
-#define NO_EXTENSION 0
-#define PRE_SHARED_KEY_EXTENSION 1
-#define KEY_SHARE_EXTENSION 2
-#define SIGNATURE_ALGORITHM_EXTENSION 4
-#define SUPPORTED_GROUPS_EXTENSION 8
-#define MAX_FRAGMENT_LENGTH_EXTENSION 16
-#define ALPN_EXTENSION 32
-#define SUPPORTED_VERSION_EXTENSION 64
-#define PSK_KEY_EXCHANGE_MODES_EXTENSION 128
-#define EARLY_DATA_EXTENSION 256
-#define SERVERNAME_EXTENSION 512
-#define COOKIE_EXTENSION 1024
-#define CID_EXTENSION 2048
+#define MBEDTLS_NO_EXTENSION                        0
```

Comment:
We need a more refined namespace here, top-level `MBEDTLS_` is too broad.

What about `MBEDTLS_SSL_EXT_ABC`

Reply:
Understand